lmcache/v1/storage_backend/naive_serde/cachegen_decoder.py

```diff
@@ -130,7 +130,7 @@ def deserialize(self, buffer_memory_obj: BytesBufferMemoryObj) -> MemoryObj:
                 dtype=kv_chunk.dtype,
                 address=-1,
                 phy_size=kv_chunk.numel() * kv_chunk.element_size(),
-                ref_count=-1,  # HACK: avoid mis-free
+                ref_count=1,  # HACK: avoid mis-free
```

Comment:
I'm not familiar with the code relevant to `cache_gen`, @YaoJiayi Would you like to take a look at this?